hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsUriQueryBuilder.java

```diff
@@ -59,6 +64,20 @@ public String toString() {
         throw new IllegalArgumentException("Query string param is not encode-able: " + entry.getKey() + "=" + entry.getValue());
       }
     }
+    // append SAS Token
+    if (sasToken != null) {
+      sasToken = sasToken.startsWith(AbfsHttpConstants.QUESTION_MARK) ?
```

Comment:
nit: move the ? down to the start ot the next line, just to make it slightly more visible